File: src/Unidesk/Unidesk.Client/src/api-client/models/FacultyDto.ts
/* istanbul ignore file */
/* tslint:disable */
/* eslint-disable */

import type { TrackedEntityDto } from './TrackedEntityDto';

export type FacultyDto = (TrackedEntityDto & {
nameEng: string;
nameCze: string;
code?: string | null;
descriptionEng?: string | null;
descriptionCze?: string | null;
});

File: src/Unidesk/Unidesk/Dtos/FacultyDto.cs
using System.ComponentModel.DataAnnotations;

namespace Unidesk.Dtos;

public class FacultyDto : TrackedEntityDto
{
    [Required]
    public string NameEng { get; set; }
    
    [Required]
    public string NameCze { get; set; }
    
    [Required]
    public string? Code { get; set; }
    
    [Required]
    public string? DescriptionEng { get; set; }
    
    [Required]
    public string? DescriptionCze { get; set; }
}